tests/query.test.js

```diff
@@ -49,6 +48,23 @@ describe('A Microstate with queries', function() {
       expect(next.completed).toHaveLength(3);
     });
   });
+
+  describe('testing find function with microstates', function () {
```

Comment:
We should be able to leave out the fact that this is a test since that is a given. My rule of thumb is that the describe + description should read comfortably a if they were a single sentence: E.g.

```js
describe("finding an element within an iterable Microstate"
  it("can locate the desired item"
  it("returns only the first result"
```

Reply:
ohh I see. thank you. that makes much more sense. will do that from now on.